pkg/operator/encryption/state.go

```diff
@@ -253,26 +250,21 @@ func encryptionConfigToEncryptionState(targetNamespace string, c *apiserverconfi
 		}
 
 		s[gr] = keysState{
-			targetNamespace: targetNamespace,
-			writeSecret:     writeSecret,
-			readSecrets:     readSecrets,
+			writeSecret: writeSecret,
+			readSecrets: readSecrets,
 		}
 	}
 	return s
 }
 
-func keyAndModeToKeySecret(targetNamespace string, k keyAndMode) (*corev1.Secret, error) {
+func keyAndModeToKeySecret(k keyAndMode) (*corev1.Secret, error) {
 	bs, err := base64.StdEncoding.DecodeString(k.key.Secret)
 	if err != nil {
 		return nil, fmt.Errorf("failed decoding base64 data of key %s", k.key.Name)
 	}
 	return &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("%s-encryption-%s", targetNamespace, k.key.Name),
-			Namespace: operatorclient.GlobalMachineSpecifiedConfigNamespace,
-			Labels: map[string]string{
-				encryptionSecretComponent: targetNamespace,
```

Comment:
We are going to list and check all the secrets in a namespace without this label. It means we might report a lot of errors for example `invalid key`. Thoughts?

Reply:
nvm, those secrets are not stored in the database.